Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp

```diff
@@ -115,6 +115,21 @@ void JitArm64::reg_imm(u32 d, u32 a, u32 value, u32 (*do_op)(u32, u32),
   }
 }
 
+u32 bit_or(u32 a, u32 b)
```

Comment:
These can be `static constexpr`. They should likely be named `BitOR`, `BitAND`, `BitXOR` to follow the coding style (ideally, [std::bit_or](http://en.cppreference.com/w/cpp/utility/functional/bit_or), [std::bit_and](http://en.cppreference.com/w/cpp/utility/functional/bit_and), and [std::bit_xor](http://en.cppreference.com/w/cpp/utility/functional/bit_xor) helpers would be used, but it's a pain in non-templated contexts for functor structs).